src/css/style.css

```diff
@@ -1,13 +1,19 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap");
 
-body {
-  padding: 0;
-  margin: 0;
-  overflow: hidden;
+/* setting css variables */
+:root {
+  --gray: #4d4d4d;
+  --light-gray: #bababa;
+  --white: #ffffff;
+  --poppins-font: Poppins, Helvetica, Arial, sans-serif;
```

Comment:
No need for this variable. We only set the font once on line 23, so it's fine to not define a variable.